CHANGES.md

```diff
@@ -5,13 +5,15 @@
 * Added a basic implementation of the draft version of OGC API - Coverages.
 * Adapted the STAC implementation to additionally offer datasets as
   individual collections for better integration with OGC API - Coverages.
+* Various minor improvements to STAC implementation.
```

Comment:
The items are so many and mostly so minor that I think it would not be helpful. Especially since STAC and Coverages are not yet in active use in production, so it's unlikely that anyone would want details of their implementation changes at this stage. I've added the full list to the issue description and added a reference to the issue in the changelog as you suggest. I've also added issue references to my other recent changelog items, which were missing them.